package.json

```diff
@@ -57,7 +57,7 @@
     "@babel/preset-env": "7.14.9",
     "@justeat/browserslist-config-fozzie": "1.2.0",
     "@justeat/eslint-config-fozzie": "5.1.0",
-    "@justeat/fozzie": "7.3.0",
+    "@justeat/fozzie": "8.0.0-beta.1",
```

Comment:
Will update this to v8 when [this PR gets merged](https://github.com/justeat/fozzie/pull/366)